backends/bmv2/midend.cpp

```diff
@@ -204,6 +204,8 @@ MidEnd::MidEnd(CompilerOptions& options) {
         new P4::TableHit(&refMap, &typeMap),
         new P4::SynthesizeActions(&refMap, &typeMap, new SkipControls(skipv1controls)),
         new P4::MoveActionsToTables(&refMap, &typeMap),
+        new P4::MidEndLast(),
+        // Here is actually the start of the BMv2-specific back-end
```

Comment:
What is "them"?
After this point there are still a few mid-end passes run, but logically what follows is bmv2-specific.